src/resmom/start_exec.c

```diff
@@ -6133,6 +6135,16 @@ start_exec(job *pjob)
 	if (do_tolerate_node_failures(pjob))
 		reliable_job_node_add(&pjob->ji_node_list, mom_host);
 
+	if (mock_run) {
```

Comment:
Not sure whether this mock run is only for non-Windows or not... but windows have different "start_exec.c" (inside src\resmom_win), maybe you want to look into that file and change if needed...

Also, (no matter whether it supported or not on windows) just make sure that this code change is not affecting on windows... as we don't run tests on Appveyor... so it better to check manually :smile:

Reply:
I didn't have plans to add support for Windows for mock run, maybe it can be taken up as a future enhancement if needed.

Appveyor does basic tests right? As long as they pass we should be good.

Reply:
Appveyor just installs msi and runs win_postinstall.py, but doesn't run any tests, no even submitting simple job as we need Linux server to do that.

Reply:
ah, ok, then ill install this on a Windows machine and do some basic checks, thanks for clarifying

Reply:
Just FYI: If you want you can get MSI from Appveyor, from the artifacts tab.